big_tests/tests/inbox_SUITE.erl

```diff
@@ -29,8 +29,11 @@
          returns_error_when_unknown_field_sent/1
         ]).
 -export([msg_sent_stored_in_inbox/1,
+         msg_sent_stored_in_inbox_queryid/1,
          msg_with_no_store_is_not_stored_in_inbox/1,
+         msg_with_no_store_is_not_stored_in_inbox_queryid/1,
          msg_with_store_hint_is_always_stored/1,
+         msg_with_store_hint_is_always_stored_queryid/1,
```

Comment:
Now, I'm not sure if all the other tests apart from `msg_sent_stored_in_inbox_queryid` are needed. Technically speaking we only want to test if the queryid is correctly used, and for that we only need one message to have been stored before sending the iq, to be sure that we'll receive _some_ message, so that we can surely test if this message does contain the queryid where it should. If there was no message stored, the check won't make much, and whether this message was stored because it contains a body or a store hint, won't matter either. Only the fact that there's a message is required.